src/test/java/com/researchspace/api/v1/controller/SamplesApiControllerMVCIT.java

```diff
@@ -448,14 +449,11 @@ public void createEditSampleWithFields() throws Exception {
   }
 
   private Optional<Sample> getComplexSampleTemplate(User user) {
-    Optional<Sample> complexTemplate =
-        sampleApiMgr.getAllTemplates(user).stream()
-            .filter(
-                t ->
-                    t.getName()
-                        .equals(ContentInitializerForDevRunManager.COMPLEX_SAMPLE_TEMPLATE_NAME))
-            .findAny();
-    return complexTemplate;
+    return sampleApiMgr.getAllTemplates(user).stream()
+        .filter(
+            t ->
+                t.getName().equals(ContentInitializerForDevRunManager.COMPLEX_SAMPLE_TEMPLATE_NAME))
```

Comment:
would it be null pointer safer to invert the parameters? like
```
                ContentInitializerForDevRunManager.COMPLEX_SAMPLE_TEMPLATE_NAME).equals(t.getName())
```

Reply:
I think `name` from `getName()` is NOT NULL in the DB, but no harm in being sure so I've swtiched them round like you suggest :+1: